app/javascript/utilities/dragAndUpload.js

```diff
@@ -0,0 +1,9 @@
+import { generateMainImage } from '../article-form/actions';
```

Comment:
When I did the refactor in #10145, I removed this file in favor of dragAndDrop.js, a new file. The new file handles only drag and drop related functionality.

The dragAndUpload is pretty much `processImageUpload` in https://github.com/forem/forem/blob/f8034bd2fd44e0e567f864d3a0f930d46d6d6a83/app/javascript/article-form/actions.js#L110 since the image has already been dropped and we're just uploading it at this point.

Consider reusing that function. It's currently in article actions, but it could be pulled out of there into another utility like imageProcessing.js. At the same time, it might not be the best abstraction.

I'll leave it up to your judgement.